lib/cubit/webhook/webhook_cubit.dart

```diff
@@ -77,8 +77,12 @@ class WebhookCubit extends Cubit<WebhookState> {
     if (lastUsedLnurlPay != null && lastUsedLnurlPay != webhookUrl) {
       await _invalidateLnurlPay(walletInfo, lastUsedLnurlPay);
     }
+    String? username = await _breezPreferences.getProfileName();
+    username = username?.replaceAll(' ', '');
     final int currentTime = DateTime.now().millisecondsSinceEpoch ~/ 1000;
-    final SignMessageRequest req = SignMessageRequest(message: '$currentTime-$webhookUrl');
+    final String optionalUsernameKey = username != null ? '-$username' : '';
+    final SignMessageRequest req =
+        SignMessageRequest(message: '$currentTime-$webhookUrl$optionalUsernameKey');
```

Comment:
I think we need a unique identifier suffix on usernames to prevent overlaps. What do you think?

Reply:
Do you mean on the default name? Like `Red Panta 21` / `redpanda21@breez.fun`?

Reply:
Yes, because the probability of two newly registered accounts sharing the same name is very high.

Also on user specified names, let's say 'Satoshi Nakamoto'.

Reply:
I think people would be annoyed if they couldn't pick their entirely own username. If it's already taken, then they have to find a different username